libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1337,6 +1337,20 @@ void ExpressionCompiler::appendArithmeticOperatorCode(Token::Value _operator, Ty
 		m_context << Instruction::DUP2 << Instruction::ISZERO;
 		m_context.appendConditionalJumpTo(m_context.errorTag());
 
+		// Test for (minInt) / (-1)
+		if (c_isSigned && _operator == Token::Div)
+		{
+			m_context << Instruction::DUP1 // x
+				<< u256(255) << u256(2) << Instruction::EXP // 2 ** 255
```

Comment:
Does the optimizer use `EXP`, `PUSH32` directly, or a much better way?  I will do an experiment.